ui/components/app/gas-timing/gas-timing.component.js

```diff
@@ -1,12 +1,65 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
+import { useGasFeeEstimates } from '../../../hooks/useGasFeeEstimates';
+import { I18nContext } from '../../../contexts/i18n';
+
 import Typography from '../../ui/typography/typography';
 import { TYPOGRAPHY } from '../../../helpers/constants/design-system';
-import InfoTooltip from '../../ui/info-tooltip';
 
-export default function GasTiming({ text, tooltipText, attitude }) {
+// Once we reach this second threshold, we switch to minutes as a unit
+const SECOND_CUTOFF = 90;
+
+export default function GasTiming({ maxPriorityFeePerGas }) {
+  const { gasFeeEstimates, isGasEstimatesLoading } = useGasFeeEstimates();
+
+  const t = useContext(I18nContext);
+
+  // Shows "seconds" as unit of time if under SECOND_CUTOFF, otherwise "minutes"
+  const toHumanReadableTime = (milliseconds = 1) => {
+    const seconds = Math.ceil(milliseconds / 1000);
+    if (seconds <= SECOND_CUTOFF) {
+      return t('gasTimingSeconds', [seconds]);
+    }
+    return t('gasTimingMinutes', [Math.ceil(seconds / 60)]);
```

Comment:
This would show both 61 seconds and 119 seconds as "2 minutes". Is that okay?

Reply:
The 61 second case would not happen, as `SECOND_CUTOFF` is 90 -- it would display as 61 seconds.

Reply:
oh, yeah, oops